src/main/resources/initDB.sql

```diff
@@ -0,0 +1,11 @@
+CREATE SCHEMA `taxi` DEFAULT CHARACTER SET utf8;
+CREATE DATABASE `taxi` /*!40100 DEFAULT CHARACTER SET utf8 */ /*!80016 DEFAULT ENCRYPTION='N' */;
```

Comment:
I'm not actually sure this will work properly, you already created 'taxi' schema. you don't need to create another DB in terms of this project